.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+---
+# https://pre-commit.com/#installation
+default_stages: [commit, push]
+default_language_version:
+  # force all unspecified Python hooks to run python3
+  python: python3
+minimum_pre_commit_version: '2.18.1'
+repos:
+  - repo: meta
+    hooks:
+      - id: identity
+      - id: check-hooks-apply
+  - repo: https://github.com/codespell-project/codespell
+    rev: v2.2.5
+    hooks:
+      - id: codespell
+        name: Run codespell
+        description: Check spelling with codespell
+        entry: codespell --ignore-words=codespell.txt
+        exclude: ^docs/image|^spark/common/src/test/resources/points\.csv$
```

Comment:
Can you please exclude all test resources in `spark/common/src/test/resources` folder?